src/components/testcase.js

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import itemTest from "../img/img_test.jpg";
```

Comment:
Is this needed?

Reply:
Not really; just for testing at that time. Deleted.